### PR TITLE
fix(quantity): disable button when min input is 0

### PR DESCRIPTION
**Pull request by skhamvon**

### CODE REVIEW

File: packages/stencil/components/quantity/src/index.html [hunk 1]
```diff
@@ -25,6 +25,14 @@
   <osds-button slot="plus" color="primary" size="sm">+</osds-button>
 </osds-quantity>
 
+<p>Quantity using osds-components (min 0)</p>
```
Comment (stevering) [inline packages/stencil/components/quantity/src/index.html:28]: :100:

File: packages/stencil/components/quantity/src/index.html [hunk 2]
```diff
@@ -25,6 +25,14 @@
...
+<osds-quantity onclick="globalMethod()" id="quantity-1">
```
Comment (stevering) [inline packages/stencil/components/quantity/src/index.html:29]: could you write a e2e test for the entire bug workflow just like you did here ?

File: packages/libraries/core/src/components/quantity/ods-quantity-controller.ts [hunk 3]
```diff
@@ -33,13 +33,15 @@ export class OdsQuantityController extends OdsComponentController<OdsQuantity> {
   }
 
   private onBlur() {
```
Comment (stevering) [inline packages/libraries/core/src/components/quantity/ods-quantity-controller.ts:35]: could you update the unit test that is testing the blur of the component with a value of 0 for min and/or max ?
and check that value is ok

File: packages/libraries/core/src/components/quantity/ods-quantity-controller.ts [hunk 4]
```diff
@@ -49,11 +51,11 @@ export class OdsQuantityController extends OdsComponentController<OdsQuantity> {
       this.logger.log("Received the customer odsValueChange event: ", event.detail);
     }
     if (this.component.input && this.component.minus && this.component.plus) {
-      if (this.component.input.value && this.component.input.min && this.component.input.max) {
-        const valueAsNb = Number(this.component.input.value);
-        const minNb = Number(this.component.input.min);
-        const maxNb = Number(this.component.input.max);
+      const valueAsNb = Number(this.component.input.value);
```
Comment (stevering) [inline packages/libraries/core/src/components/quantity/ods-quantity-controller.ts:54]: could you update te unit test for this change ?

**Review by stevering — CHANGES_REQUESTED** [review]
almost !
just some update of tests todo i think

File: packages/libraries/core/src/components/quantity/ods-quantity-controller.ts (hunk 3 above)
Comment (skhamvon) [reply packages/libraries/core/src/components/quantity/ods-quantity-controller.ts:35]: There is no test for onBlur since it is private

**Comment by ovh-cds** [comment]
CDS Report build-storybook#275.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#275.1 ✘  
* 
  * build ✘

**Review by stevering — APPROVED** [review]

**Pull request merged**